docs/dev_guide/migrate-a-project/generate-compilation-db.rst

```diff
@@ -115,6 +115,20 @@ in the build directory:
 
    CMake 3.10 or higher is recommended as it provides better support for CUDA source code.
 
+Generate a Compilation Database for Setuptools-Based Projects
+--------------------------------------------------------
+Setuptools is a package development library in Python for facilitating packaging, distribution and installation of Python projects.For projects with setuptools as their build system, the steps to generate a compilation database are as follows:
+
+#. Generate the build log for the project using the verbose flag 
+#. Use ``intercept-build –-parse-build-log`` option to generate the compilation database.
+
+Example:
+
+.. code-block:: none
+
+    python3 setup.py build_ext -v > build.log 2>&1

```

Comment:
Are option "-v" and "build_ext" general options for setup.py?

Reply:
build_ext is used to build extension module, and -v is the verbose flag. I have mentioned an example command here so it is easy to understand. Do you suggest I should change it?

Reply:
Suggest: python3 setup.py <build_option> -v > build.log 2>&1

Reply:
Done